packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -1,49 +0,0 @@
-/* tslint:disable */
```

Comment:
I know this wasn't really exported, but to reduce the chance that someone relied on this, keep these files around and re-export the renamed versions as their original named things. That way it reduces the chance of their being a break in someone consuming this.

Reply:
If we do this, we should do it for all the removed old namings... and in the case of `SuggestionItemDefault` it's impossible as they are incompatible now.

One thing to make sure we are not breaking anyone is to just have the old files along with their supporting `.scss` files still exist and later with a major bump removed completely. Would these be considered a good solution?

Reply:
I think that would probably be fine for right now.